src/constants.ts

```diff
@@ -31,3 +31,9 @@ export namespace AppServiceDialogResponses {
 }
 
 export const envFileName: string = '.env';
+
+export enum Platform {
```

Comment:
Yeah I've been meaning to get rid of this constant in Functions. The intellisense/validation are a lot better than I remember in the past:
![Screen Shot 2019-11-07 at 2 14 40 PM](https://user-images.githubusercontent.com/11282622/68432312-0d9fed00-0169-11ea-9a79-bf3556ab58be.png)
![Screen Shot 2019-11-07 at 2 15 11 PM](https://user-images.githubusercontent.com/11282622/68432313-0e388380-0169-11ea-9ef3-bb8708cf4de8.png)